### PR TITLE
:sparkles: Adicionando feature para validar formato de email

### PR DESCRIPTION
**Pull request by Leonardo18**

### CODE REVIEW

File: packages/is-valid-email/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "brazilian",
+      "utils",
+      "validador",
+      "validator"
+    ],
+    "license": "MIT",
+    "devDependencies": {
+      "microbundle": "^0.6.0"
+    },
+    "dependencies": {
```
Comment (hyanmandian) [inline packages/is-valid-email/package.json]: Remove these dependencies

File: .all-contributorsrc [hunk 2]
```diff
@@ -86,6 +86,17 @@
...
+        {
+      "login": "Leonardo18",
+      "name": "Leonardo Dutra",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/14338574?v=4",
+      "profile": "https://github.com/Leonardo18",
+      "contributions": [
+        "code",
+        "doc",
+        "test"
+      ]
```
Comment (hyanmandian) [inline .all-contributorsrc]: Update README.md with `yarn contributors:generate`

Comment (Leonardo18) [reply .all-contributorsrc]: Updated

File: packages/is-valid-email/package.json (hunk 1 above)
Comment (Leonardo18) [reply packages/is-valid-email/package.json]: Updated

**Pull request merged**